metadata-ingestion/src/datahub/ingestion/source/unity/source.py

```diff
@@ -517,17 +521,29 @@ def process_table(self, table: Table, schema: Schema) -> Iterable[MetadataWorkUn
                 yield from self.gen_siblings_workunit(dataset_urn, source_dataset_urn)
                 yield from self.gen_lineage_workunit(dataset_urn, source_dataset_urn)
 
+        if ownership:
+            patch_builder = create_dataset_owners_patch_builder(dataset_urn, ownership)
+
+            for patch_mcp in patch_builder.build():
+                yield MetadataWorkUnit(
+                    id=f"{dataset_urn}-{patch_mcp.aspectName}", mcp_raw=patch_mcp
+                )
+
+        patch_builder = create_dataset_props_patch_builder(dataset_urn, table_props)
+        for patch_mcp in patch_builder.build():
+            yield MetadataWorkUnit(
+                id=f"{dataset_urn}-{patch_mcp.aspectName}", mcp_raw=patch_mcp
+            )
+
```

Comment:
Do we need check for table_props existence similar to ownership?

Reply:
I don't think so, because if we don't have `include_ownership: true` in the recipe file, we are not generating the `ownership` and in that case the value would be `None` and hence the check is in place.

For the `table_props` we definitely having values like `createdAt, createdBy` and that will not be `None`; hence the check is not in place; but I will add an additional check here